packages/clay-card/src/Detail.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+interface CardDetailProps extends React.HTMLAttributes<HTMLDivElement> {
+	children: React.ReactNode;
+}
+
+export type CardDetailType = React.FunctionComponent<CardDetailProps>;
+
+export const Detail: CardDetailType = ({
+	children,
+	className,
+	...otherProps
+}) => {
+	const {displayType} = React.useContext(Context);
+	const isCardType = {
+		horizontalInteractive: displayType === 'horizontal interactive',
```

Comment:
Having two different values passed into display type is pretty flakey. I think it'd be better to go with two props of `horizontal` and `interactive`.

Reply:
Solved

Reply:
Solved.